seed/analysis_pipelines/better/pipeline.py

```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
         ExtraDataColumnPath(
             'better_cost_savings_combined',
             'BETTER Potential Cost Savings (USD)',
+            1,
             'assessment.assessment_energy_use.cost_savings_combined'
         ),
         ExtraDataColumnPath(
             'better_energy_savings_combined',
             'BETTER Potential Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_combined'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_combined',
-            'BETTER Potential GHG Emissions Reduction (kgCO2e)',
+            'BETTER Potential GHG Emissions Reduction (MTCO2e)',
+            .001,
             'assessment.assessment_energy_use.ghg_reductions_combined'
         ),
         # Energy-specific Savings
         ExtraDataColumnPath(
             BETTER_VALID_MODEL_E_COL,
             'BETTER Valid Electricity Model',
+            1,
             'assessment.assessment_energy_use.valid_model_e'
         ),
         ExtraDataColumnPath(
             BETTER_VALID_MODEL_F_COL,
             'BETTER Valid Fuel Model',
+            1,
             'assessment.assessment_energy_use.valid_model_f'
         ),
         ExtraDataColumnPath(
             'better_cost_savings_electricity',
             'BETTER Potential Electricity Cost Savings (USD)',
+            1,
             'assessment.assessment_energy_use.cost_savings_e'
         ),
         ExtraDataColumnPath(
             'better_cost_savings_fuel',
             'BETTER Potential Fuel Cost Savings (USD)',
+            1,
             'assessment.assessment_energy_use.cost_savings_f'
         ),
         ExtraDataColumnPath(
             'better_energy_savings_electricity',
             'BETTER Potential Electricity Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_e'
         ),
         ExtraDataColumnPath(
             'better_energy_savings_fuel',
             'BETTER Potential Fuel Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_f'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_electricity',
             'BETTER Potential Electricity GHG Emissions Reduction (kgCO2e)',
```

Comment:
we'll need to convert this to tons co2e, but we can do that in another PR.

Reply:
Actually, should that happen in this PR, since you just created the multiplier argument?

Reply:
It should, yes.  Updating!